<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MIBLR-LP0015" ignored="0" name="DentalBatchTopay" tests="2" failures="1" timestamp="2021-04-01T20:18:55 IST" time="70.783" errors="0">
  <testcase name="moveToEFTPaymentFromBatchToPay" time="23.705" classname="com.adjudication.test.viewclaims.dental.batchtopay.MoveToEFTPaymentFromDental">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at com.adjudication.test.viewclaims.dental.batchtopay.MoveToEFTPaymentFromDental.moveToEFTPaymentFromBatchToPay(MoveToEFTPaymentFromDental.java:103)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- moveToEFTPaymentFromBatchToPay -->
  <testcase name="moveToPaidFromEFTPaymentDental" time="23.756" classname="com.adjudication.test.viewclaims.dental.batchtopay.MoveToEFTPaymentFromDental"/>
</testsuite> <!-- DentalBatchTopay -->
